### DISCUSSION TITLE
Is reordering a `GridView` using drag-and-drop supported? (on Android, iOS, and WinUI)

### DISCUSSION THREAD

**Discussion by jeanplevesque** [unoplatform/uno#12938] [Q&A]
We currently use drag-and-drop reordering with `ListView` and we're curious whether it's supported with `GridView` because we would like to use it to improve some user experiences.

**Comment by dr1rrb** [comment]
Support of re-ordering is done in the `ListViewBase`, so theoretically it should work ... but I never tested :/
